JASP-Engine/JASP/R/common.R

```diff
@@ -3074,3 +3074,9 @@ postProcessModuleInstall <- function(moduleLibraryPath)
   #             "The argument options should be the options list from QML.",
   #             "Ensure that the SetSeed{} QML component is present in the QML file for this analysis."))
 }
+
+.getSeedJASP <- function(options) {
```

Comment:
Unit tests found a syntax error -- one `{` is not closed

Reply:
ok, thanks. Fixed